resources/artifact-ignores.properties

```diff
@@ -809,3 +809,4 @@ xframium = https://www.jenkins.io/security/plugins/#suspensions
 
 # https://jenkins.io/security/advisory/2022-11-15/
 config-rotator = https://www.jenkins.io/security/plugins/#suspensions
+lightstep-incident-response-plugin = https://github.com/jenkinsci/lightstep-incident-response-plugin/blob/main/README.md
```

Comment:
```suggestion
lightstep-incident-response = https://github.com/jenkinsci/lightstep-incident-response-plugin/blob/main/README.md
```

Please do not ignore review feedback when opening a successor PR: https://github.com/jenkins-infra/update-center2/pull/665#discussion_r1047855084

Reply:
cam someone review this please?

Reply:
@daniel-beck Gentle Reminder.

Reply:
@sn-priya-jagyasi you still need to replace `lightstep-incident-response-plugin` with `lightstep-incident-response` as suggested both here and in the original PR.